tls/extensions/s2n_client_status_request.c

```diff
@@ -32,3 +32,19 @@ int s2n_extensions_client_status_request_send(struct s2n_connection *conn, struc
 
     return 0;
 }
+
+int s2n_recv_client_status_request(struct s2n_connection *conn, struct s2n_stuffer *extension)
+{
+    if (s2n_stuffer_data_available(extension) < 5) {
```

Comment:
Define a constant value for `5` here.

Reply:
This is just moving the logic over. from one file to another. removal of magic numbers will come at a later point in optimizing